src/uct/sm/base/sm_iface.h

```diff
@@ -8,12 +8,19 @@
 #define SM_IFACE_H_
 
 #include <uct/api/uct.h>
+#include <uct/base/uct_iface.h>
 #include <ucs/sys/math.h>
 #include <ucs/sys/sys.h>
 
 #define UCT_SM_IFACE_DEVICE_ADDR_LEN    sizeof(uint64_t)
 #define UCT_SM_MAX_IOV                  16
 
+extern ucs_config_field_t uct_sm_iface_common_config_table[];
+
+typedef struct uct_sm_iface_common_config {
```

Comment:
inherit from uct_iface_config?

Reply:
i'll not be able to use common config table